sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -693,6 +693,16 @@ def test_ordinal_encoder_raise_missing(X):
         ohe.transform(X)
 
 
+def test_ordinal_encoder_raise_categories_shape():
+
+    X = np.array([['Low', 'Medium', 'High', 'Medium', 'Low']], dtype=object).T
+    cats = ['Low', 'Medium', 'High']
+    enc = OrdinalEncoder(categories=cats)
+    msg = ("Shape mismatch: if categories is an array,")
+    #       "Call 'fit' with appropriate arguments before using this method.")
```

Comment:
What is the reason for this comment? (I don't really understand it)

Reply:
My bad.. that was unrelated. Have removed it now.